src/App.vue

```diff
@@ -110,6 +110,14 @@ export default {
       await this.reloadUserData();
     },
   },
+  metaInfo() {
+    return {
+      // default title
+      title: 'Superhero.com',
+      // will be injected into page meta
+      titleTemplate: `${this.$route.meta.title} | Superhero.com`,
```

Comment:
```suggestion
      titleTemplate: `${this.$route.meta.title} - Superhero.com`,
```
be consistent with the previous implementation, I don't want to change the title format now

Reply:
right